src/quo2/components/settings/accounts/view.cljs

```diff
@@ -0,0 +1,40 @@
+(ns quo2.components.settings.accounts.view
```

Comment:
@ulisesmac, we have also started to *attempt* to write "component tests" for new components in `quo2/`. It would be very welcomed if you could test some things.

You can find examples in `__tests__` directories. Documentation about testing components is on the way, but right now the best is to just stick to the existing examples in the codebase.

Reply:
Ok, I'll take a look at those examples and think in some tests. I'll reply when done

Reply:
Feel free to share any problems you might find. `make component-test-watch` should be enough to get you going.

- Example test: `src/quo2/components/selectors/filter/component_spec.cljs`
- You need to add the new test namespace to `src/quo2/core_spec.cljs`.

Reply:
@ilmotta I've already had a look at those tests, but I'm not clear how to write them, also I'm not familiar with `testing-library/react-native` yet.
So I'd like to wait for the documentation on how to write & run tests in order to write my own.

Could this MR be accepted even if it doesn't have tests?

Reply:
Understandable @ulisesmac, we're not blocking PRs lacking such tests. There's definitely a learning curve to write them effectively.